examples/simple_regional_beta/main.tf

```
@@ -51,6 +51,10 @@ module "gke" {
   # Disable workload identity
   identity_namespace = null
   node_metadata      = "UNSPECIFIED"
+
+  # Support Dataplane Setup
+  network_policy    = var.network_policy
+  datapath_provider = var.datapath_provider
```

nit: We can hardcode these in examples instead of plumbing from fixture.
```suggestion
  datapath_provider = "ADVANCED_DATAPATH"
```